datafusion/core/tests/fuzz_cases/sort_fuzz.rs

```diff
@@ -136,9 +136,9 @@ impl SortTest {
                     .sort_spill_reservation_bytes,
             );
 
-            let runtime_config = RuntimeConfig::new()
-                .with_memory_pool(Arc::new(GreedyMemoryPool::new(pool_size)));
-            let runtime = Arc::new(RuntimeEnv::new(runtime_config).unwrap());
+            let runtime_env = RuntimeConfig::new()
+                .with_memory_pool(Arc::new(GreedyMemoryPool::new(pool_size))).build();
+            let runtime = Arc::new(runtime_env.unwrap());
```

Comment:
we should avoid calling `.unwrap()` although its not related to the PR

Reply:
Sure. Even for tests, if we have complex chains of unwraps, we can use `expect` with a more detailed failure explanation.

Reply:
(note this is a test so I think it is ok to call unwrap() here)